Dockerfile.temporal

```diff
@@ -0,0 +1,23 @@
+FROM temporalio/auto-setup:1.20
+
+# Set up environment variables for PostgreSQL and Elasticsearch
+ENV DB=postgresql \
```

Comment:
The PostgreSQL credentials (username and password) are set to `temporal` which is not secure for production use.
There's no explicit setting for `POSTGRES_DB` which might cause issues if the database name is expected to be different from the default.